modules/serving-HA-components.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-config-HA.adoc
+
+[id="serving-HA-components_{context}"]
+= Knative Serving high availability components
```

Comment:
The headings/TOC seem a little off on this. I'd suggest instead having a concept module and then a procedure module, at the same level. Something like:

```
= About Knative Serving high availability
= Disabling high availability
```